workflow-core/src/commonMain/kotlin/com/squareup/workflow1/WorkerWorkflow.kt

```diff
@@ -89,8 +89,9 @@ private class EmitWorkerOutputAction<P, S, O>(
   private val output: O,
 ) : WorkflowAction<P, S, O>() {
   override fun toString(): String =
-    WorkflowIdentifierTypeNamer.uniqueName(EmitWorkerOutputAction::class) +
-      "(worker=$worker, key=\"$renderKey\")"
+    WorkflowIdentifierTypeNamer.uniqueName(EmitWorkerOutputAction::class)
+  // TODO tmp Removed because this creates noise.
+  // + "(worker=$worker, key=\"$renderKey\")"
```

Comment:
output would look like this:

```
sendAndAwaitApplication(com.squareup.workflow1.EmitWorkerOutputAction(worker=TypedWorker(java.lang.String (Kotlin reflection is not available)), key="1"))
```

key was useful, `worker=TypedWorker(java.lang.String (Kotlin reflection is not available))` definitely not useful. Really wish we'd get rid of these `Kotlin reflection is not available` in technical names btw.

The real reason I removed this is because `key="1"` has `"` and I needed to escape those in the output for the graph and was lazy so I just removed them.

Reply:
We should probably remove the `" "` I don't think we use those anywhere else.

Reply:
> Really wish we'd get rid of these Kotlin reflection is not available in technical names btw.

Yes! They're horrible, but I've lost track of where they're sneaking in. If you find the source, please let me know and I'll jump on it.